lib/options/dcu.robot

```diff
@@ -48,3 +48,29 @@ Get UEFI Option
     ...    cd dcu && ./dcu v coreboot.rom --get "${option_name}"
     ...    shell=True
     RETURN    ${out.stdout}
+
+Execute Warmboot Command
+    [Documentation]    Executes a command that will cause a warmboot
+
+    # Using "Execute Command In Terimal" will cause the test to wait
+    # for command prompt to appear before continuing but the prompt
+    # will not appear again until we Login after reboot, so the test
+    # would hang here and fail.
+    # Sometimes it may take long to shutdown all systemd services,
+    # so the waiting times have to be excessive to avoid false negatives.
+    Write Into Terminal    rtcwake -m off -s 20
+    Set DUT Response Timeout    300s
+    Sleep    20s
+
+Execute Suspend And Wake Command
+    [Documentation]    Suspends and then wakes up the device after some time
+
+    # Using "Execute Command In Terimal" will cause the test to wait
+    # for command prompt to appear before continuing but the prompt
+    # will not appear again until we Login after reboot, so the test
+    # would hang here and fail.
+    # Sometimes it may take long to shutdown all systemd services,
+    # so the waiting times have to be excessive to avoid false negatives.
+    Write Into Terminal    rtcwake -m disk -s 20
```

Comment:
```suggestion
    Write Into Terminal    rtcwake -m disk -s 20
```

Suspend to disk is hibernation, the suspend tests typically test suspend to RAM (S3) and / or suspend to idle (S0ix). I'd use `mem` here

Reply:
Done

Reply:
Thanks. Please remember to link to commits that address each review comment

Reply:
Oh, sure, it would be this one [50dd4f4](https://github.com/Dasharo/open-source-firmware-validation/pull/344/commits/50dd4f4a09dc87ef61ac4b4684e45300c2e0d23d). This commit addressed two comments. Next time i will address every comment in a seperate commit.